e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js

```diff
@@ -82,6 +82,81 @@ describe('I18456 Built-in slash commands: common', () => {
+        loginAndVisitDefaultChannel(user1, testChannelUrl);
+
+        const usernames1 = Cypress._.map(userGroup, 'username').slice(0, 4);
+        const usernames1Format = [
+
+            // # Format for sending a group message:
+            // /groupmsg @[username1],@[username2],@[username3] [message]
+            `@${usernames1[0]}, @${usernames1[1]}, @${usernames1[2]}, @${usernames1[3]}`,
+
+            // # Use /groupmsg command and use a mix of @ symbols in front of some names but not all
+            // # Format notes:
+            // # Usernames do not have to contain the '@' character
+            // # Accepts spaces after or before the commas when listing usernames
+            `${usernames1[0]}, @${usernames1[1]} , ${usernames1[2]} , @${usernames1[3]}`,
+        ];
+
+        usernames1Format.forEach((users) => {
+            // # Use /groupmsg command to send group message - "/groupmsg [usernames] [message]"
+            mesg = MESSAGES.SMALL;
+            const command = `/groupmsg ${users} ${mesg}`;
+            cy.postMessage(command);
+
+            // * Sends a Group Message to the specified users
+            cy.uiWaitUntilMessagePostedIncludes(mesg);
+            cy.getLastPostId().then((postId) => {
+                cy.get(`#postMessageText_${postId}`).should('have.text', mesg);
+            });
+            usernames1.forEach((username) => {
+                cy.contains('.channel-header__top', username).should('be.visible');
+            });
+
+            cy.contains('.sidebar-item', 'Town Square').click();
+        });
+
+        usernames1Format.forEach((users) => {
+            // # Use /groupmsg command to send message to existing GM - "group msg [usernames]" (note: no message)
+            // # Format notes: The command does not have to contain a message
+            const command = `/groupmsg ${users}`;
+            cy.postMessage(command);
+
+            // * Message sent to existing GM as expected
+            cy.uiWaitUntilMessagePostedIncludes(mesg);
+            cy.getLastPostId().then((postId) => {
+                cy.get(`#postMessageText_${postId}`).should('have.text', mesg);
+            });
+            usernames1.forEach((username) => {
+                cy.contains('.channel-header__top', username).should('be.visible');
+            });
+
+            cy.contains('.sidebar-item', 'Town Square').click();
+        });
+
+        const usernames2 = Cypress._.map(userGroup, 'username').slice(1, 5);
+        const usernames2Format = [
+            `@${usernames2[0]}, @${usernames2[1]}, @${usernames2[2]}, @${usernames2[3]}`,
+            `${usernames2[0]}, @${usernames2[1]} , ${usernames2[2]} , @${usernames2[3]}`,
+        ];
+
+        usernames2Format.forEach((users) => {
+            // # Use /groupmsg command to create GM - "group msg [usernames]" (note: no message)
+            // # Format notes: The command does not have to contain a message
+            const command = `/groupmsg ${users}{enter}`;
```

Comment:
For some reasons, `{enter}` ensures the postMessage command hit the enter key so that the test does not fail intermittently. It is only necessary in the last part of the test. I can't figure out why.

Reply:
Works for me without it but I'm good to leave it as is. We'll just monitor once merged.